app/src/main/res/values/strings.xml

```diff
@@ -547,17 +547,17 @@
     <string name="import_youtube_instructions">Import YouTube subscriptions from Google takeout:\n\n1. Go to this URL: %1$s\n2. Log in when asked\n3. Click on \"All data included\", then on \"Deselect all\", then select only \"subscriptions\" and click \"OK\"\n4. Click on \"Next step\" and then on \"Create export\"\n5. Click on the \"Download\" button after it appears and \n6. From the downloaded takeout zip extract the .json file (usually under \"YouTube and YouTube Music/subscriptions/subscriptions.json\") and import it here.</string>
     <string name="import_soundcloud_instructions">Import a SoundCloud profile by typing either the URL or your ID:\n\n1. Enable \"desktop mode\" in a web-browser (the site is not available for mobile devices)\n2. Go to this URL: %1$s\n3. Log in when asked\n4. Copy the profile URL you were redirected to.</string>
     <string name="import_soundcloud_instructions_hint">yourID, soundcloud.com/yourid</string>
-    <string name="import_network_expensive_warning">Keep in mind this operation can be network expensive.\n\nDo you want to continue?</string>
+    <string name="import_network_expensive_warning">Continue even though it may use a lot of network data?</string>
     <!-- Playback Parameters -->
-    <string name="playback_speed_control">Playback Speed Controls</string>
+    <string name="playback_speed_control">Controls for speed</string>
```

Comment:
Changing this string to what you are using is also changing strings' sense. Please revert your changes on it.

Reply:
```suggestion
    <string name="playback_speed_control">Playback Speed Controls</string>
```